tests/integration/base.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         UNSAFE_ENABLE_ADMIN_ENDPOINTS: true
         PILOT_REMOTE_CLUSTER_TIMEOUT: 15s
         PILOT_ENABLE_ALPHA_GATEWAY_API: "true"
+        PILOT_MX_ADDITIONAL_LABELS: "custom-label"
```

Comment:
nit: can you set it in `tests/integration/telemetry/api/setup_test.go` -- this applies to every test, we only need it on the test we are using it IMO.

Else we have no coverage of NOT having this feature set.

Reply:
good catch, I forgot I can do that per suite.